docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
        - .env
     volumes:
       - ./proxy-template.conf:/etc/nginx/templates/default.conf.template:ro
+      - ./common.conf:/etc/nginx/templates/common.conf.template:ro
```

Comment:
Lets create a subdirectory with common and nginx template file and then we can use /etc/nginx/templates/ to pass all the files with one volut

Reply:
I agree.